test/unit_tests/modules/util/type-checkers-spec.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+var chai = require( 'chai' );
+var expect = chai.expect;
+var typeCheckers = require(
+  '../../../../src/static/js/modules/util/type-checkers.js'
+);
+
+var undefinedVar;
+var blankVar = '';
+var aString = 'bar';
+var aNum = 42;
+var aDate = new Date( 2011, 7, 21 );
+function aFunction() {
+  return true;
+}
+var anObject = {
+  a: '1',
+  b: '2',
+  c: '3'
+};
+var anArray = [ 1, 2, 3 ];
+
+describe( 'TypeCheckers isUndefined', function() {
+  it( 'should identify undefined variables', function() {
+    expect( typeCheckers.isUndefined( undefinedVar ) ).to.be.true;
```

Comment:
They're both comparable. I don't know if we'd be able to use jsdom for mocking with Jasmine, but the main reason we went with Mocha is that it's what Owning a Home, eRegs and a couple other projects are using.

http://thejsguy.com/2015/01/12/jasmine-vs-mocha-chai-and-sinon.html